opam

```diff
@@ -1,19 +0,0 @@
-
```

Comment:
I was thinking that we should check in this file, so that when we tag a build and generate a tgz file for opam it matches exactly with the check in.